android/test_probes/widgets/probe_button.py

```diff
@@ -0,0 +1,8 @@
+from java import jclass
```

Comment:
A slightly bikeshed comment: but how would you feel about:

* Renaming `test_probes/probe_button.py::ButtonProbe` to `probes/button_probe.py::ButtonProbe`
* Adding `from .button_probe import ButtonProbe` to `probes/__init__.py`.

That way:
1. The namespace collision issue with copying test code isn't an problem any more
2. There's consistency of naming between `button_probe.py` and `ButtonProbe` (FWIW, ˆ still prefer for `probes/button.py`, even if there is multiple button.py files; but I can live with a longer name if it's at least consistent)
3. It's clear about isolating "test instrumentation" from "test code"
4. The usage pattern in tests will be `from probes import ButtonProbe`.

Reply:
Now that we have a few examples in place, I think you were right that it's better to keep the common test code and the backend-specific code in separate namespaces. So since we already have `tests`, I've gone with `tests_backend`, to mirror the relationship between `toga` and `toga_cocoa` etc., and because there will probably be things other than probes in there soon enough.

And I've removed the `probe_` prefix on the module names as you suggest.